card-game-client/app/components/ui/game-card.tsx

```diff
@@ -95,16 +112,54 @@ export default function GameCard({
 
   const handleSave = () => {
     // add save logic here
-    console.log("Saving edited options:", editedOptions);
+    console.log("Saving new card:", {
+      title: editedTitle,
+      category: editedCategory,
+      question: editedQuestion,
+      options: editedOptions,
+      imageUrl,
+    });
     onClose?.();
   };
 
+  const handleImageUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
```

Comment:
Is this function only support single file upload?

Reply:
Yes, do we want multiple file upload?

Reply:
NOPE